pkg/kapp/diffgraph/change_graph_test.go

```diff
@@ -465,6 +469,73 @@ metadata:
 	}
 }
 
+func TestChangeGraphWithChangeGroupPlaceholderInConfDefaults(t *testing.T) {
```

Comment:
Looking for a better name

Reply:
TestChangeGraphWithChangeGroupPlaceholderInConfDefaults -> TestChangeGraphWithNamespaceAndCRDs

Reply:
Done